lib/obsolete.gi

```diff
@@ -891,5 +891,46 @@ end);
 
 #############################################################################
 ##
+#F  SetUserPreferences
+##
+##  Set the defaults of `GAPInfo.UserPreferences'.
+##
+##  We locate the first file `gap.ini' in GAP root directories,
+##  and read it if available.
+##  This must be done before `GAPInfo.UserPreferences' is used.
+##  Some of the preferences require an initialization,
+##  but this cannot be called before the complete library has been loaded.
+##
+BindGlobal( "SetUserPreferences", function( arg )
+    local name, record;
+
+    Info( InfoWarning, 1, "");
+    Info( InfoWarning, 1, Concatenation( [
```

Comment:
Shouldn't this use `InfoObsolete` (I know you just moved it, but this would be a logical change to apply next, wouldn't it?